### PR TITLE
fix for #13623 by hiding Member Edit Action Menu on Infinite editing …

### LINKED ISSUE #13623: Infinite Editors and Dialogs conflict
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

11.0.0

### Bug summary

When editing a Member in infinite editor, then choosing an action for that Member that brings up a dialog, the dialog is unusable.

Case in point:
Selecting a Member that is associated with a content item brings it up in the Infinite editor.  If I try to delete the member using the Actions menu, the controls are obscured:

delete-dialog-issue1

This can also be tested by installing the MemberListView plugin (version 3.0.0 recently released):

delete-dialog-issue2

It's interesting to note that Members in the Infinite Editor have an Actions menu, while Content Items do not.  Perhaps it should be disabled if this issue can't be fixed?

### Specifics

This happens from Umbraco 8.8.15 right through to Umbraco 11.

### Steps to reproduce

1. Install MemberListView (https://www.nuget.org/packages/MemberListView/) into a new Umbaco installation.
2. Create a new member, then attempt to edit it via the MemberListView
3. Attempt to delete the member from the Actions menu on the top right.

### Expected result / actual result

Either the dialog should become fully interactive when it shows up on top of the infinite editor, or the actions menu for members should be hidden as it is for Content items.

### PR DESCRIPTION
**Pull request by robertjf**
…for consistency

### Prerequisites

- [x] I have added steps to test this contribution in the description below

If there's an existing issue for this PR then this fixes 

### Description
This fixes #13623 by hiding the Action Menu when the Member is edited in the Infinite Editor.  This brings it in line with the experience of editing Media or Content items in the Infinite Editor.

For steps to reproduce, see ticket #13623

### CODE REVIEW

**Review by mikecp — CHANGES_REQUESTED** [review]
Hi @robertjf,

Thanks for spotting this issue and providing the fix  👍

Before merging, may I ask you to add the trailing ";" at the end of the code you added 😁? It might sound picky but it's best to stay inline with the rest of the code I think 😅

Also, could you maybe quickly check the behavior for @Zeegaan 's scenario as described in #13623? On my pc the action menu is still visible in that case, so I'm wondering if it's just me...

Thanks and cheers !!

**Comment by robertjf** [comment]
@mikecp oh of course! Not sure how I missed that... will get it fixed :)

**Comment by Zeegaan** [comment]
Tests out great 🎉 I will let @mikecp merge this if he is happy with the code 😁

**Review by mikecp — APPROVED** [review]

**Comment by mikecp** [comment]
Thanks @robertjf for the quick check/fix, it works great now 🎉
So let's merge this, as @Zeegaan suggests 😁

Cheers and (almost) Happy New Year !! 🎆

**Comment by nul800sebastiaan** [comment]
Cherry picked for 10.5 in 4b2af296be06093890eee3dad3612d0dc16d261d

**Pull request merged**
